Automattic/wp-calypso
# Email: Create email provider comparison page

#### Changes proposed in this Pull Request

This adds the email provider comparison page, replacing the G Suite upsell when trying to add email for a domain. Right now this is purely UI with some minor business logic like deciding the pricing to show for G Suite. More functionality will be added in a separate PR.

#### Testing instructions

* Load Calypso with the `email/titan-mvp` flag enabled (append  this querystring`?flags=email/titan-mvp` to the URL)
* Go to domains
* Click on the "Add" button under the "Email" column for a domain
* Verify you see something like the screenshot below

P.S. the buttons don't work, so please focus on the UI and the code.

<img width="1054" alt="Email Provider Comparison UI" src="https://user-images.githubusercontent.com/13062352/92302316-90e9c500-ef6b-11ea-80f5-f78051aebce8.png">
